client/app/components/packages/rwcds-form/with-action-no-action.hbs

```diff
@@ -0,0 +1,186 @@
+{{#let @form as |Form|}}
+  <Form.Section @title="Analysis">
+    <Ui::Question as |Q|>
+      <Q.Prompt>
+        Describe the development assumptions. These assumption could be the result of existing conditions, local policies, etc.
+      </Q.Prompt>
+      <Ui::CollapsibleText
+        @buttonText="Additional Guidance"
+      >
+        <p>
+          Consider the guidance on soft site criteria provided in the CEQR Technical Manual Chapter, "Establishing the Analysis Framework" (pages 2-6 and 2-7, 2-10 and 2-11), in conjunction with neighborhood-specific trends in the "Study Area."
+        </p>
+        <p>
+          Summarize the criteria used to identify soft sites while including the following information:
+        </p>
+          <ul>
+            <li>Rationale for selecting Projected Development Sites and Potential Development Sites</li>
+            <li>Possible use conversions</li>
+            <li>Expansions or sites with bulk or uses that would be legalized in the future with the Proposed Actions</li>
+            <li>Other Sites that may be excluded from the analysis, if applicable (e.g., rent stabilized    residential buildings, which are difficult to legally demolish due to tenant relocation requirements). Please note that dwelling unit sizes typically range between 850 – 1,000 gsf.</li>
+          </ul>
+        <p>Please note that dwelling unit sizes typically range between 850 – 1,000 gsf.</p>
+      </Ui::CollapsibleText>
+      <Form.Field
+        @attribute="dcpDevelopmentsiteassumptions"
+        @type="text-area"
+        @maxlength="2400"
+      />
+    </Ui::Question>
+
+    <h3 class="xlarge-margin-top">
+      No-Action Scenario
+    </h3>
+    <Ui::Question as |Q|>
```

Comment:
If the question is a radio group, it should use `@fieldset={{true}}`, so it's a fieldset and its prompt is a legend. Otherwise, the whole Q will be wrapped in a label, which will cause the prompt to be clickable.